src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -51,10 +51,10 @@ public ProcessMetrics(ProcessInstrumentationOptions options)
             "process.memory.virtual",
             () =>
             {
-                return Diagnostics.Process.GetCurrentProcess().PrivateMemorySize64;
+                return Diagnostics.Process.GetCurrentProcess().VirtualMemorySize64;
             },
             unit: "By",
-            description: "The amount of virtual memory allocated for this process that cannot be shared with other processes.");
+            description: "All the address space this process can read from without trigerring an access violation.");
```

Comment:
```suggestion
            description: "The amount of committed virtual memory for this process.");
```
I think this is how most developers and tools would refer to it. I know it doesn't match the exact definition of committed memory Mark Russinovich gave in that blog post, but I think people will be more confused if we don't call it committed memory :)